### PR TITLE
feat: add file adapter

### PR DESCRIPTION
**Pull request by Zxilly**
Signed-off-by: Zxilly <zhouxinyu1001@gmail.com>

### CODE REVIEW

**Comment by Zxilly** [comment]
@nodece @hsluoyz plz review

File: .gitignore [hunk 1]
```diff
@@ -1,104 +1,7 @@
-# Logs
```
Comment (hsluoyz) [inline .gitignore:1]: Don't remove

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Comment by hsluoyz** [comment]
@Zxilly setup README, badges and semantic-release

**Comment by Zxilly** [comment]
@hsluoyz I will first add unit tests

**Pull request merged**